addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -353,7 +389,9 @@ var ListController = BasicController.extend({
         // we prevent the event propagation because we don't want this event to
         // trigger a click on the main bus, which would be then caught by the
         // list editable renderer and would unselect the newly created row
-        event.stopPropagation();
+        if (event) {
```

Comment:
why is it needed? if the method is sometimes called without an event, this code is not a handler and should be moved in a private function, and the handler will call it